services/dgxie/values.yaml

```diff
@@ -0,0 +1,51 @@
+# Default values for dgxie.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# DGXie config
+bootMode: DGX
+isoPath: /iso
+dgxNetInt: enp1s0f0
+dgxDisk: sda
+dgxKbd: us
+dgxKernExtra: ''
+mgmtIntPub: eth0
+mgmtIntPrv: eth1
+netDomain: local
+netPrvIp: 192.168.1.1
+netPrvNet: 192.168.1.0
+netPrvNetmask: 255.255.255.0
+netPrvGateway: 192.168.1.1
+netPrvDns1: 192.168.1.1
+netPrvDns2: 8.8.8.8
+netPrvDhcpStart: 192.168.1.100
+netPrvDhcpEnd: 192.168.1.199
+netPrvDhcpLease: 7200
+httpPort: 13370
+
+# Deployment config
+replicaCount: 1
+
+image:
+  repository: dgxie
```

Comment:
should this be `registry.local/deepops/dgxie`?

Reply:
currently you build the dgxie container locally and copy it over to the mgmt host and load it in via docker, so it's not coming from the registry. it probably would be a better idea to bootstrap the registry and put the container there, but this config correct for now

Reply:
👍